components/LineGraph/LineGraph.js

```diff
@@ -88,7 +88,7 @@ class LineGraph extends Component {
     this.initialRender();
   }
 
-  componentWillReceiveProps(nextProps) {
```

Comment:
WillREceiveProps gets called before ShouldComponentUpdate, which defeats the purpose of this PR, so changed it to componentWillUpdate, which comes after